control_allocation:
  thruster_right_pos: [-2.373776, -1.027135, 0.318237]
  thruster_left_pos: [-2.373776, 1.027135, 0.318237]
trajectory_tracker:
  gains:
    kp_x: 40.0
    ki_x: 0.01
    kd_x: 50.0
    kp_y: 20.0
    ki_y: 0.01
    kd_y: 25.0
    anti_windup_x: .25
    anti_windup_y: .25
  init_state:
    x_init: 0.0
    y_init: 0.0
    psi_init: 0.0
    xdot_init: 0.0
    ydot_init: 0.0
    psidot_init: 0.0
  init_commanded:
    xc_init: 0.0
    yc_init: 0.0
    psic_init: 0.0
    xdotc_init: 0.0
    ydotc_init: 0.0
    psidotc_init: 0.0
heading_control:
  kp: 100.0
  ki: 0.05
  kd: 15.0
Ts: 0.1
